### PR TITLE
Add a 2022q2 report for ZFS support in makefs

### PR DESCRIPTION
**Pull request by markjdb**

### CODE REVIEW

File: 2022q2/makefs-zfs.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Contact: Mark Johnston <markj@FreeBSD.org>
+
+makefs(8) is a utility that creates file system images entirely in userspace.
+It is a useful component of a toolchain to build virtual machine (VM) images
+since it does not require any special privileges, unlike the approach of formatting
+a character device, mounting the fresh file system, and copying files onto it.
+Moreover, makefs can create reproducible images and aims to minimize
+resource consumption.
+Currently makefs can build UFS, cd9660, and msdos (FAT) file system images.
```
Comment (emaste) [inline 2022q2/makefs-zfs.adoc:15]: might want "FreeBSD's makefs" or such, as NetBSD upstream has other filesystems that we don't care about

File: 2022q2/makefs-zfs.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+images.
+
+Some recent work enables the creation of ZFS images by makefs.
+This makes it easier to build ZFS-based VM images.
+makefs' ZFS support includes the ability to create multiple datasets, with each
+mapped to a directory in the input file hierarchy.
+Many ZFS features are not supported however, as the implementation provides only
+what is needed to get reproducible root pools.
+
+Some follow-up work enables the creation of ZFS-based VM and cloud images by
```
Comment (pauamma) [inline 2022q2/makefs-zfs.adoc:26]: s/Some follow-up work/Follow-up work/ (same as above).

If, as it seems to me, you haven't started yet, I'd also s/enables/will enable/.

File: 2022q2/makefs-zfs.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+entirely in userspace.
+It is a useful component of a toolchain to build virtual machine (VM) images
+since it does not require any special privileges, unlike the approach of formatting
+a character device, mounting the fresh file system, and copying files onto it.
+Moreover, makefs can create reproducible images and aims to minimize
+resource consumption.
+Currently, FreeBSD's makefs can build UFS, cd9660, and msdos (FAT) file system
+images.
+
+Some recent work enables the creation of ZFS images by makefs.
```
Comment (pauamma) [inline 2022q2/makefs-zfs.adoc:19]: s/Some recent work/Recent work/ (indefinite works as partitive here and reads better IMO).

**Review by pauamma — CHANGES_REQUESTED** [review]
Next time, could you avoid force-pushing, per https://github.com/freebsd/freebsd-quarterly#readme  item 5?

File: 2022q2/makefs-zfs.adoc (hunk 2 above)
Comment (markjdb) [reply 2022q2/makefs-zfs.adoc]: It is started, I added a review link.

**Review by pauamma — APPROVED** [review]

**Pull request merged**